src/components/tooltip/README.md

```diff
@@ -245,13 +249,22 @@ prop modifier.
       return {
         disabled: false
       }
+    },
+    methods: {
+      disableByRoot() {
+        if (this.disabled){
+          this.$root.$emit('bv::enable::popover')
```

Comment:
these emit events need the trigger button's ID string (the id of the button that has the tooltip), otherwise it enables or disables **all** tooltips

Also the event should be `bv::{enable|disable}::tooltip` for tooltips 😉

Reply:
Yep, true